sdk/tables/Azure.Data.Tables/src/BatchItem.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Data.Tables
+{
+    /// <summary>
+    ///
+    /// </summary>
+    public class BatchItem
+    {
+        /// <summary>
+        ///
+        /// </summary>
+        public BatchOperation BatchOperation { get; set; }
```

Comment:
Interesting... so items in a batch can request different operations?

Reply:
An entity can only be represented once per batch, but yes on any operation.